docs/source/index.md

```diff
@@ -100,6 +100,14 @@ features/compatibility_matrix
 
 % Details about running vLLM
 
+:::{toctree}
+:caption: Training
```

Comment:
`Training` --> `RLHF`  or `Reinforcement Learning` ?

Reply:
I initially went with Training because it's possible for users to use vLLM for evals in SFT (which we might write about in future). What do you think?

Reply:
"Training" is fairly general and can encompass any type of training, without specifying the underlying method.

Reply:
at the first glance, i thought it is "training resource for learning vllm" lol